src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/CharSetSolver.cs

```diff
@@ -78,13 +69,7 @@ public BDD RangeConstraint(char c, char d, bool ignoreCase = false, string? cult
                 return CharConstraint(c, ignoreCase, culture);
```

Comment:
Same here... once CharConstraint is modified to no longer take ignoreCase and culture, RangeConstraint can similiarly have those parameters removed?